sigstore/_internal/trustroot.py

```diff
@@ -117,20 +288,36 @@ def _get_ca_keys(
             for cert in ca.cert_chain.certificates:
                 yield cert.raw_bytes
 
-    def get_ctfe_keys(self) -> list[bytes]:
+    def rekor_keyring(self) -> RekorKeyring:
+        """Return public key contents given certificate authorities."""
+
+        return RekorKeyring(self._get_rekor_keys())
```

Comment:
I believe you can just remove this wrapper method, and change 
 `_get_rekor_keys(self) -> Keyring`
 into
 `rekor_keyring(self) -> RekorKeyring`
right?

Reply:
it makes sense